Fixed Query that tried to get non existen attributes, improved error …

Sina-CAU opened the pull request:
…logging for queries

dependabot[bot] committed:
chore(deps): bump serialize-javascript in /frontend-nx

Bumps [serialize-javascript](https://github.com/yahoo/serialize-javascript) from 6.0.0 to 6.0.2.
- [Release notes](https://github.com/yahoo/serialize-javascript/releases)
- [Commits](https://github.com/yahoo/serialize-javascript/compare/v6.0.0...v6.0.2)

---
updated-dependencies:
- dependency-name: serialize-javascript
  dependency-type: indirect
...

Signed-off-by: dependabot[bot] <support@github.com>

Sina-CAU committed:
Fixed Query that tried to get non existen attributes, improved error logging for queries

Sina-CAU committed:
Fixed Query that tried to get non existen attributes, improved error logging for queries

steffen74 reviewed (changes requested):
@Sina-CAU 
You change the generic table grid component here and seem to add characteristics specific to just the anonymization.

dependabot[bot] committed:
Bump @babel/runtime from 7.20.6 to 7.26.10 in /frontend-nx

Bumps [@babel/runtime](https://github.com/babel/babel/tree/HEAD/packages/babel-runtime) from 7.20.6 to 7.26.10.
- [Release notes](https://github.com/babel/babel/releases)
- [Changelog](https://github.com/babel/babel/blob/main/CHANGELOG.md)
- [Commits](https://github.com/babel/babel/commits/v7.26.10/packages/babel-runtime)

---
updated-dependencies:
- dependency-name: "@babel/runtime"
  dependency-type: indirect
...

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
Bump tj-actions/changed-files from 41 to 46 in /.github/workflows

Bumps [tj-actions/changed-files](https://github.com/tj-actions/changed-files) from 41 to 46.
- [Release notes](https://github.com/tj-actions/changed-files/releases)
- [Changelog](https://github.com/tj-actions/changed-files/blob/main/HISTORY.md)
- [Commits](https://github.com/tj-actions/changed-files/compare/v41...v46)

---
updated-dependencies:
- dependency-name: tj-actions/changed-files
  dependency-type: direct:production
...

Signed-off-by: dependabot[bot] <support@github.com>

jeremy-oc committed:
Add messageKey to return dict in load_participation_data function

jeremy-oc committed:
Merge pull request #1299 from eduhub-org/bugfix/participant-data-csv-export-apolloerror

Add messageKey to return dict in load_participation_data function

jeremy-oc committed:
Add Plausible Analytics

steffen74 committed:
Merge pull request #1307 from eduhub-org/feature/add-plausible-analytics

Feature/add plausible analytics

steffen74 committed:
Merge pull request #1294 from eduhub-org/dependabot/github_actions/dot-github/workflows/tj-actions/changed-files-46

Bump tj-actions/changed-files from 41 to 46 in /.github/workflows

dependabot[bot] committed:
Bump next from 14.2.21 to 14.2.26 in /frontend-nx

Bumps [next](https://github.com/vercel/next.js) from 14.2.21 to 14.2.26.
- [Release notes](https://github.com/vercel/next.js/releases)
- [Changelog](https://github.com/vercel/next.js/blob/canary/release.js)
- [Commits](https://github.com/vercel/next.js/compare/v14.2.21...v14.2.26)

---
updated-dependencies:
- dependency-name: next
  dependency-version: 14.2.26
  dependency-type: direct:production
...

Signed-off-by: dependabot[bot] <support@github.com>

steffen74 committed:
Merge pull request #1310 from eduhub-org/dependabot/npm_and_yarn/frontend-nx/next-14.2.26

Bump next from 14.2.21 to 14.2.26 in /frontend-nx

steffen74 committed:
Merge pull request #1287 from eduhub-org/dependabot/npm_and_yarn/frontend-nx/babel/runtime-7.26.10

Bump @babel/runtime from 7.20.6 to 7.26.10 in /frontend-nx

steffen74 committed:
Merge pull request #1233 from eduhub-org/dependabot/npm_and_yarn/frontend-nx/serialize-javascript-6.0.2

chore(deps): bump serialize-javascript from 6.0.0 to 6.0.2 in /frontend-nx

steffen74 committed:
updates  localhost to hasura in graphql URI definition of UpdateFromKeycloak to avoid IPv6 confusion

dependabot[bot] committed:
Bump http-proxy-middleware from 2.0.7 to 2.0.9 in /frontend-nx

Bumps [http-proxy-middleware](https://github.com/chimurai/http-proxy-middleware) from 2.0.7 to 2.0.9.
- [Release notes](https://github.com/chimurai/http-proxy-middleware/releases)
- [Changelog](https://github.com/chimurai/http-proxy-middleware/blob/v2.0.9/CHANGELOG.md)
- [Commits](https://github.com/chimurai/http-proxy-middleware/compare/v2.0.7...v2.0.9)

---
updated-dependencies:
- dependency-name: http-proxy-middleware
  dependency-version: 2.0.9
  dependency-type: indirect
...

Signed-off-by: dependabot[bot] <support@github.com>

steffen74 committed:
Add standardite tableGridDeletionButton with confirmation in manage course and programs
Fixes #1313

steffen74 committed:
catches addional foreign key violation for user deletoin

steffen74 committed:
Merge pull request #1314 from eduhub-org/issue1313/Add-deletion-confirmation-in-mage-course-and-programs

Issue1313/add deletion confirmation in mage course and programs

steffen74 committed:
updates ordering of the organizations to start as a default with the newest

steffen74 committed:
changes the learning goals to read only for instructors

steffen74 committed:
removes unecessary legacy code for deleting programs
improves translations

steffen74 committed:
removes unecessary imports

steffen74 committed:
Merge pull request #1311 from eduhub-org/dependabot/npm_and_yarn/frontend-nx/http-proxy-middleware-2.0.9

Bump http-proxy-middleware from 2.0.7 to 2.0.9 in /frontend-nx

steffen74 committed:
adds functionality to correctly set times for new session creations

steffen74 committed:
corrects default start and end time for new sessions to have the correct format
improves initial setting for session times
adds standard components for setting session times

steffen74 committed:
deletes not needed imports (removes warnings)

steffen74 committed:
improves commenting for date time helper functions

steffen74 committed:
removes unused call and components

steffen74 committed:
fixes time zone problems for setting the course start and end

Sina-CAU committed:
Remaining error message, Users can be deleted

Sina-CAU committed:
Merge branch 'bugfix/users-cant-be-deleted' of https://github.com/eduhub-org/eduhub into bugfix/users-cant-be-deleted

Sina-CAU committed:
Tried to fix the error messages (404 Keycloak), but they still remain
